code/game/objects/items/implants/implantuplink.dm

```diff
@@ -10,6 +10,16 @@
 /obj/item/implant/uplink/Initialize(mapload, _owner)
 	. = ..()
 	AddComponent(/datum/component/uplink, _owner, TRUE, FALSE, null, starting_tc)
+	RegisterSignal(src, COMSIG_COMPONENT_REMOVING, .proc/_component_removal)
+
+/obj/item/implant/uplink/proc/_component_removal(datum/source, datum/component/component)
+	/**

```

Comment:
This documentation is incorrectly executed, the documentation should be on the line above the proc (line 14 in this current version)

[More information on our documentation requirements can be found here](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)